cpp/oneapi/dal/algo/decision_forest/backend/gpu/train_feature_type_dpc.cpp

```diff
@@ -34,8 +35,15 @@ inline sycl::event sort_inplace(sycl::queue& queue_,
                                 pr::ndarray<Float, 1>& src,
                                 const bk::event_vector& deps = {}) {
     ONEDAL_ASSERT(src.get_count() > 0);
+    auto device = queue_.get_device();
+    std::string device_name = device.get_info<sycl::info::device::name>();
     auto src_ind = pr::ndarray<Index, 1>::empty(queue_, { src.get_count() });
-    return pr::radix_sort_indices_inplace<Float, Index>{ queue_ }(src, src_ind, deps);
+    if (device_name.find("Data Center GPU Max") != std::string::npos) {
+        return pr::radix_sort_indices_inplace_dpl<Float, Index>(queue_, src, src_ind, deps);
```

Comment:
are there issues on other GPUs? and should we be validating CI beyond default private scope to run bazel tests on it?

Reply:
only pvc for now supports onedpl sort. It will be automatically dispatched in our CI